// _@vuepress_core@1.8.2@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/home/jack/文档/00前端代码/myblog/node_modules/_@vuepress_core@1.8.2@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-f31b7fb8",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f31b7fb8").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-6c7ff374",
    path: "/blogs/01Arithmetic/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6c7ff374").then(next)
    },
  },
  {
    path: "/blogs/01Arithmetic/index.html",
    redirect: "/blogs/01Arithmetic/"
  },
  {
    name: "v-5faf5571",
    path: "/blogs/01Arithmetic/03set.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5faf5571").then(next)
    },
  },
  {
    name: "v-05e4fc52",
    path: "/blogs/01Arithmetic/02js_datatruce.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-05e4fc52").then(next)
    },
  },
  {
    name: "v-2c7f9352",
    path: "/blogs/01Arithmetic/01datastruce.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2c7f9352").then(next)
    },
  },
  {
    name: "v-620dd9ff",
    path: "/blogs/02Font_end/01-VSCode.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-620dd9ff").then(next)
    },
  },
  {
    name: "v-829d88d2",
    path: "/blogs/02Font_end/02html.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-829d88d2").then(next)
    },
  },
  {
    name: "v-59db8b34",
    path: "/blogs/02Font_end/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-59db8b34").then(next)
    },
  },
  {
    path: "/blogs/02Font_end/index.html",
    redirect: "/blogs/02Font_end/"
  },
  {
    name: "v-2e762946",
    path: "/blogs/02Font_end/03html5.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2e762946").then(next)
    },
  },
  {
    name: "v-a47cda2a",
    path: "/blogs/03python/01Variablesanddatatypes.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a47cda2a").then(next)
    },
  },
  {
    name: "v-a76b6b16",
    path: "/blogs/03python/02Typeconversionandoperator.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a76b6b16").then(next)
    },
  },
  {
    name: "v-d2c7b552",
    path: "/blogs/03python/04String.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d2c7b552").then(next)
    },
  },
  {
    name: "v-617edeef",
    path: "/blogs/03python/03Loopstructure.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-617edeef").then(next)
    },
  },
  {
    name: "v-5d4981f7",
    path: "/blogs/03python/05lists_tuples.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5d4981f7").then(next)
    },
  },
  {
    name: "v-3a6ec86b",
    path: "/blogs/03python/06Dictionariesandsets.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3a6ec86b").then(next)
    },
  },
  {
    name: "v-6ad2ef12",
    path: "/blogs/03python/07CommonOperations.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6ad2ef12").then(next)
    },
  },
  {
    name: "v-9c35fe12",
    path: "/blogs/03python/08derivated.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-9c35fe12").then(next)
    },
  },
  {
    name: "v-fd9bc326",
    path: "/blogs/03python/09FundamentalsofFunctions.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-fd9bc326").then(next)
    },
  },
  {
    name: "v-e7cb90d2",
    path: "/blogs/03python/10Functionadvanced.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-e7cb90d2").then(next)
    },
  },
  {
    name: "v-0311329e",
    path: "/blogs/03python/11packets.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0311329e").then(next)
    },
  },
  {
    name: "v-7498f517",
    path: "/blogs/03python/12AnonymousFunctions.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7498f517").then(next)
    },
  },
  {
    name: "v-45c8057e",
    path: "/blogs/03python/13Advancedbuilt-infunctions.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-45c8057e").then(next)
    },
  },
  {
    name: "v-ed7a7752",
    path: "/blogs/03python/14file.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ed7a7752").then(next)
    },
  },
  {
    name: "v-ec57fc18",
    path: "/blogs/03python/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ec57fc18").then(next)
    },
  },
  {
    path: "/blogs/03python/index.html",
    redirect: "/blogs/03python/"
  },
  {
    name: "v-4d4d4137",
    path: "/blogs/04python_web/02multitasking.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4d4d4137").then(next)
    },
  },
  {
    name: "v-694655f7",
    path: "/blogs/04python_web/01Networkprogramming.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-694655f7").then(next)
    },
  },
  {
    name: "v-4b6b1b34",
    path: "/blogs/04python_web/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4b6b1b34").then(next)
    },
  },
  {
    path: "/blogs/04python_web/index.html",
    redirect: "/blogs/04python_web/"
  },
  {
    name: "v-517a93dd",
    path: "/docs/learnThink/01JavaScript_let_const_var.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-517a93dd").then(next)
    },
  },
  {
    name: "v-3b337897",
    path: "/blogs/04python_web/03multitasking.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3b337897").then(next)
    },
  },
  {
    name: "v-48275004",
    path: "/docs/learnThink/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-48275004").then(next)
    },
  },
  {
    path: "/docs/learnThink/index.html",
    redirect: "/docs/learnThink/"
  },
  {
    name: "v-5a647cba",
    path: "/docs/personSource/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5a647cba").then(next)
    },
  },
  {
    path: "/docs/personSource/index.html",
    redirect: "/docs/personSource/"
  },
  {
    name: "v-b1564aac",
    path: "/tag/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tags", "v-b1564aac").then(next)
    },
    meta: {"pid":"tags","id":"tags"}
  },
  {
    path: "/tag/index.html",
    redirect: "/tag/"
  },
  {
    name: "v-ef9325c4",
    path: "/categories/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("FrontmatterKey", "v-ef9325c4").then(next)
    },
    meta: {"pid":"categories","id":"categories"}
  },
  {
    path: "/categories/index.html",
    redirect: "/categories/"
  },
  {
    name: "v-6319eb4e",
    path: "/timeline/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("TimeLines", "v-6319eb4e").then(next)
    },
    meta: {"pid":"timeline","id":"timeline"}
  },
  {
    path: "/timeline/index.html",
    redirect: "/timeline/"
  },
  {
    name: "v-89abd2d6",
    path: "/tag/算法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-89abd2d6").then(next)
    },
    meta: {"pid":"tags","id":"算法"}
  },
  {
    path: "/tag/算法/index.html",
    redirect: "/tag/算法/"
  },
  {
    name: "v-97019ce8",
    path: "/tag/工具/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-97019ce8").then(next)
    },
    meta: {"pid":"tags","id":"工具"}
  },
  {
    path: "/tag/工具/index.html",
    redirect: "/tag/工具/"
  },
  {
    name: "v-49a56e2a",
    path: "/tag/前端/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-49a56e2a").then(next)
    },
    meta: {"pid":"tags","id":"前端"}
  },
  {
    path: "/tag/前端/index.html",
    redirect: "/tag/前端/"
  },
  {
    name: "v-f1650aa2",
    path: "/tag/HTML5/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-f1650aa2").then(next)
    },
    meta: {"pid":"tags","id":"HTML5"}
  },
  {
    path: "/tag/HTML5/index.html",
    redirect: "/tag/HTML5/"
  },
  {
    name: "v-9499bfde",
    path: "/tag/python基础/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-9499bfde").then(next)
    },
    meta: {"pid":"tags","id":"python基础"}
  },
  {
    path: "/tag/python基础/index.html",
    redirect: "/tag/python基础/"
  },
  {
    name: "v-3229d5f6",
    path: "/tag/web/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-3229d5f6").then(next)
    },
    meta: {"pid":"tags","id":"web"}
  },
  {
    path: "/tag/web/index.html",
    redirect: "/tag/web/"
  },
  {
    name: "v-309cc62e",
    path: "/tag/python/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-309cc62e").then(next)
    },
    meta: {"pid":"tags","id":"python"}
  },
  {
    path: "/tag/python/index.html",
    redirect: "/tag/python/"
  },
  {
    name: "v-d52b2726",
    path: "/tag/学习总结/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-d52b2726").then(next)
    },
    meta: {"pid":"tags","id":"学习总结"}
  },
  {
    path: "/tag/学习总结/index.html",
    redirect: "/tag/学习总结/"
  },
  {
    name: "v-054c7780",
    path: "/categories/01算法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-054c7780").then(next)
    },
    meta: {"pid":"categories","id":"01算法"}
  },
  {
    path: "/categories/01算法/index.html",
    redirect: "/categories/01算法/"
  },
  {
    name: "v-7b69a574",
    path: "/categories/02前端/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-7b69a574").then(next)
    },
    meta: {"pid":"categories","id":"02前端"}
  },
  {
    path: "/categories/02前端/index.html",
    redirect: "/categories/02前端/"
  },
  {
    name: "v-41d86ee3",
    path: "/categories/03HTML/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-41d86ee3").then(next)
    },
    meta: {"pid":"categories","id":"03HTML"}
  },
  {
    path: "/categories/03HTML/index.html",
    redirect: "/categories/03HTML/"
  },
  {
    name: "v-1aa94cf1",
    path: "/categories/04python/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-1aa94cf1").then(next)
    },
    meta: {"pid":"categories","id":"04python"}
  },
  {
    path: "/categories/04python/index.html",
    redirect: "/categories/04python/"
  },
  {
    name: "v-d2128d04",
    path: "/categories/05web/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-d2128d04").then(next)
    },
    meta: {"pid":"categories","id":"05web"}
  },
  {
    path: "/categories/05web/index.html",
    redirect: "/categories/05web/"
  },
  {
    name: "v-624401fe",
    path: "/categories/06JavaScript/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-624401fe").then(next)
    },
    meta: {"pid":"categories","id":"06JavaScript"}
  },
  {
    path: "/categories/06JavaScript/index.html",
    redirect: "/categories/06JavaScript/"
  },
  {
    name: "v-32d21ee8",
    path: "/tag/python基础/page/2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-32d21ee8").then(next)
    },
    meta: {"pid":"tags","id":"python基础"}
  },
  {
    path: "/tag/python基础/page/2/index.html",
    redirect: "/tag/python基础/page/2/"
  },
  {
    name: "v-8cc2f0a8",
    path: "/categories/04python/page/2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-8cc2f0a8").then(next)
    },
    meta: {"pid":"categories","id":"04python"}
  },
  {
    path: "/categories/04python/page/2/index.html",
    redirect: "/categories/04python/page/2/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
