// _@vuepress_core@1.8.2@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/yaya/Documents/blog/Doerr33.github.io/node_modules/_@vuepress_core@1.8.2@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-7bfc5141",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7bfc5141").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-30f26a5a",
    path: "/blogs/01Arithmetic/01datastruce.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-30f26a5a").then(next)
    },
  },
  {
    name: "v-7fdd725e",
    path: "/blogs/01Arithmetic/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7fdd725e").then(next)
    },
  },
  {
    path: "/blogs/01Arithmetic/index.html",
    redirect: "/blogs/01Arithmetic/"
  },
  {
    name: "v-180da5c2",
    path: "/blogs/02Font_end/01-VSCode.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-180da5c2").then(next)
    },
  },
  {
    name: "v-0d997ab4",
    path: "/blogs/01Arithmetic/03set.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0d997ab4").then(next)
    },
  },
  {
    name: "v-382ffc5a",
    path: "/blogs/01Arithmetic/02js_datatruce.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-382ffc5a").then(next)
    },
  },
  {
    name: "v-149b05ae",
    path: "/blogs/03python/01Variablesanddatatypes.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-149b05ae").then(next)
    },
  },
  {
    name: "v-d2a1dec0",
    path: "/blogs/02Font_end/03html5.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d2a1dec0").then(next)
    },
  },
  {
    name: "v-46b7909c",
    path: "/blogs/03python/03Loopstructure.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-46b7909c").then(next)
    },
  },
  {
    name: "v-d89b6b90",
    path: "/blogs/03python/02Typeconversionandoperator.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d89b6b90").then(next)
    },
  },
  {
    name: "v-0c44bf5a",
    path: "/blogs/03python/04String.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0c44bf5a").then(next)
    },
  },
  {
    name: "v-34642f7a",
    path: "/blogs/03python/05lists_tuples.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-34642f7a").then(next)
    },
  },
  {
    name: "v-24be77b0",
    path: "/blogs/03python/09FundamentalsofFunctions.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-24be77b0").then(next)
    },
  },
  {
    name: "v-3459d59a",
    path: "/blogs/02Font_end/02html.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3459d59a").then(next)
    },
  },
  {
    name: "v-05b902fa",
    path: "/blogs/03python/07CommonOperations.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-05b902fa").then(next)
    },
  },
  {
    name: "v-71869bcc",
    path: "/blogs/03python/10Functionadvanced.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-71869bcc").then(next)
    },
  },
  {
    name: "v-782f1511",
    path: "/blogs/02Font_end/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-782f1511").then(next)
    },
  },
  {
    path: "/blogs/02Font_end/index.html",
    redirect: "/blogs/02Font_end/"
  },
  {
    name: "v-43c3d6a4",
    path: "/blogs/03python/06Dictionariesandsets.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-43c3d6a4").then(next)
    },
  },
  {
    name: "v-a73ce818",
    path: "/blogs/03python/11packets.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a73ce818").then(next)
    },
  },
  {
    name: "v-76f805f8",
    path: "/blogs/03python/13Advancedbuilt-infunctions.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-76f805f8").then(next)
    },
  },
  {
    name: "v-7f8e23fa",
    path: "/blogs/04python_web/01Networkprogramming.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7f8e23fa").then(next)
    },
  },
  {
    name: "v-0d24761a",
    path: "/blogs/03python/12AnonymousFunctions.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0d24761a").then(next)
    },
  },
  {
    name: "v-f4edfb5e",
    path: "/blogs/03python/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f4edfb5e").then(next)
    },
  },
  {
    path: "/blogs/03python/index.html",
    redirect: "/blogs/03python/"
  },
  {
    name: "v-ef20888c",
    path: "/blogs/04python_web/02multitasking.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ef20888c").then(next)
    },
  },
  {
    name: "v-67e4ccba",
    path: "/blogs/03python/08derivated.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-67e4ccba").then(next)
    },
  },
  {
    name: "v-7655f31a",
    path: "/blogs/04python_web/03multitasking.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7655f31a").then(next)
    },
  },
  {
    name: "v-c20722de",
    path: "/blogs/04python_web/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-c20722de").then(next)
    },
  },
  {
    path: "/blogs/04python_web/index.html",
    redirect: "/blogs/04python_web/"
  },
  {
    name: "v-38e293a0",
    path: "/docs/learnThink/01JavaScript_let_const_var.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-38e293a0").then(next)
    },
  },
  {
    name: "v-833a95ca",
    path: "/docs/learnThink/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-833a95ca").then(next)
    },
  },
  {
    path: "/docs/learnThink/index.html",
    redirect: "/docs/learnThink/"
  },
  {
    name: "v-31fd0357",
    path: "/docs/personSource/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-31fd0357").then(next)
    },
  },
  {
    path: "/docs/personSource/index.html",
    redirect: "/docs/personSource/"
  },
  {
    name: "v-65f6d7da",
    path: "/blogs/03python/14file.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-65f6d7da").then(next)
    },
  },
  {
    name: "v-b1564aac",
    path: "/tag/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tags", "v-b1564aac").then(next)
    },
    meta: {"pid":"tags","id":"tags"}
  },
  {
    path: "/tag/index.html",
    redirect: "/tag/"
  },
  {
    name: "v-ef9325c4",
    path: "/categories/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("FrontmatterKey", "v-ef9325c4").then(next)
    },
    meta: {"pid":"categories","id":"categories"}
  },
  {
    path: "/categories/index.html",
    redirect: "/categories/"
  },
  {
    name: "v-6319eb4e",
    path: "/timeline/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("TimeLines", "v-6319eb4e").then(next)
    },
    meta: {"pid":"timeline","id":"timeline"}
  },
  {
    path: "/timeline/index.html",
    redirect: "/timeline/"
  },
  {
    name: "v-89abd2d6",
    path: "/tag/算法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-89abd2d6").then(next)
    },
    meta: {"pid":"tags","id":"算法"}
  },
  {
    path: "/tag/算法/index.html",
    redirect: "/tag/算法/"
  },
  {
    name: "v-97019ce8",
    path: "/tag/工具/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-97019ce8").then(next)
    },
    meta: {"pid":"tags","id":"工具"}
  },
  {
    path: "/tag/工具/index.html",
    redirect: "/tag/工具/"
  },
  {
    name: "v-9499bfde",
    path: "/tag/python基础/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-9499bfde").then(next)
    },
    meta: {"pid":"tags","id":"python基础"}
  },
  {
    path: "/tag/python基础/index.html",
    redirect: "/tag/python基础/"
  },
  {
    name: "v-f1650aa2",
    path: "/tag/HTML5/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-f1650aa2").then(next)
    },
    meta: {"pid":"tags","id":"HTML5"}
  },
  {
    path: "/tag/HTML5/index.html",
    redirect: "/tag/HTML5/"
  },
  {
    name: "v-49a56e2a",
    path: "/tag/前端/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-49a56e2a").then(next)
    },
    meta: {"pid":"tags","id":"前端"}
  },
  {
    path: "/tag/前端/index.html",
    redirect: "/tag/前端/"
  },
  {
    name: "v-3229d5f6",
    path: "/tag/web/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-3229d5f6").then(next)
    },
    meta: {"pid":"tags","id":"web"}
  },
  {
    path: "/tag/web/index.html",
    redirect: "/tag/web/"
  },
  {
    name: "v-309cc62e",
    path: "/tag/python/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-309cc62e").then(next)
    },
    meta: {"pid":"tags","id":"python"}
  },
  {
    path: "/tag/python/index.html",
    redirect: "/tag/python/"
  },
  {
    name: "v-d52b2726",
    path: "/tag/学习总结/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-d52b2726").then(next)
    },
    meta: {"pid":"tags","id":"学习总结"}
  },
  {
    path: "/tag/学习总结/index.html",
    redirect: "/tag/学习总结/"
  },
  {
    name: "v-054c7780",
    path: "/categories/01算法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-054c7780").then(next)
    },
    meta: {"pid":"categories","id":"01算法"}
  },
  {
    path: "/categories/01算法/index.html",
    redirect: "/categories/01算法/"
  },
  {
    name: "v-7b69a574",
    path: "/categories/02前端/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-7b69a574").then(next)
    },
    meta: {"pid":"categories","id":"02前端"}
  },
  {
    path: "/categories/02前端/index.html",
    redirect: "/categories/02前端/"
  },
  {
    name: "v-1aa94cf1",
    path: "/categories/04python/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-1aa94cf1").then(next)
    },
    meta: {"pid":"categories","id":"04python"}
  },
  {
    path: "/categories/04python/index.html",
    redirect: "/categories/04python/"
  },
  {
    name: "v-41d86ee3",
    path: "/categories/03HTML/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-41d86ee3").then(next)
    },
    meta: {"pid":"categories","id":"03HTML"}
  },
  {
    path: "/categories/03HTML/index.html",
    redirect: "/categories/03HTML/"
  },
  {
    name: "v-d2128d04",
    path: "/categories/05web/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-d2128d04").then(next)
    },
    meta: {"pid":"categories","id":"05web"}
  },
  {
    path: "/categories/05web/index.html",
    redirect: "/categories/05web/"
  },
  {
    name: "v-624401fe",
    path: "/categories/06JavaScript/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-624401fe").then(next)
    },
    meta: {"pid":"categories","id":"06JavaScript"}
  },
  {
    path: "/categories/06JavaScript/index.html",
    redirect: "/categories/06JavaScript/"
  },
  {
    name: "v-32d21ee8",
    path: "/tag/python基础/page/2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-32d21ee8").then(next)
    },
    meta: {"pid":"tags","id":"python基础"}
  },
  {
    path: "/tag/python基础/page/2/index.html",
    redirect: "/tag/python基础/page/2/"
  },
  {
    name: "v-8cc2f0a8",
    path: "/categories/04python/page/2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-8cc2f0a8").then(next)
    },
    meta: {"pid":"categories","id":"04python"}
  },
  {
    path: "/categories/04python/page/2/index.html",
    redirect: "/categories/04python/page/2/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
