// _@vuepress_core@1.8.2@@vuepress/core/.temp/dynamic/vuepress_blog/paginations.js
/**
 * Generated by "@vuepress/plugin-blog"
 */

import sorters from './pageSorters'
import filters from './pageFilters'

export default [{
  pid: "tags",
  id: "算法",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/算法/","interval":[0,4]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "工具",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/工具/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "HTML5",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/HTML5/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "前端",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/前端/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "python基础",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/python基础/","interval":[0,9]},{"path":"/tag/python基础/page/2/","interval":[10,15]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "web",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/web/","interval":[0,4]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "python",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/python/","interval":[0,4]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "学习总结",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/学习总结/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "01算法",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/01算法/","interval":[0,4]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "03HTML",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/03HTML/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "02前端",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/02前端/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "04python",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/04python/","interval":[0,9]},{"path":"/categories/04python/page/2/","interval":[10,19]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "05web",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/05web/","interval":[0,4]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "06JavaScript",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/06JavaScript/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
}]
